apollo-router-core/src/layers/opentracing.rs

```diff
@@ -0,0 +1,133 @@
+use std::fmt::Display;
+use std::task::{Context, Poll};
+
+use crate::layer::ConfigurableLayer;
+use crate::{register_layer, services, SubgraphRequest};
+use http::HeaderValue;
+use opentelemetry::trace::TraceContextExt;
+use schemars::JsonSchema;
+use serde::Deserialize;
+use tower::{BoxError, Layer, Service};
+use tracing::instrument::Instrumented;
+use tracing::{span, Instrument, Level, Span};
+use tracing_opentelemetry::OpenTelemetrySpanExt;
+
+register_layer!("opentracing", OpenTracingLayer);
```

Comment:
Suggest we don't register this layer and instead we configure it via the reporting plugin